### PR TITLE
HTML5 - Whiteboard/Presentation fixes, improvements, refactoring

### PR DESCRIPTION
**Pull request by OZhurbenko**
Most of the changes are based on the #4345 reviews.
- Whiteboard-Toolbar refactoring
- Added and started using another config file for the whiteboard
- Minor corrections / small code changes
- Moved all `resize` events needed by the whiteboard to the HoC

Plus a couple of bug fixes:
- Fixed a bug with the slide pushing the whiteboard-toolbar out of the window (when you narrow it) in multi-user mode in Firefox
- Fixed inconsistencies between initial whiteboard-toolbar display values and actual values
- Prevented exceptions thrown when the slide file was not accessible in a long non-active meeting

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/notifications-bar/component.jsx [hunk 1]
```diff
@@ -1,45 +1,36 @@
...
 import cx from 'classnames';
+import injectWbResizeEvent from '/imports/ui/components/presentation/resize-wrapper/component';
 import styles from './styles.scss';
 
 const COLORS = [
   'default', 'primary', 'danger', 'success',
 ];
 
 const propTypes = {
+  children: PropTypes.string.isRequired,
```
[the remark this answers is not in the document]
Comment (OZhurbenko) [reply bigbluebutton-html5/imports/ui/components/notifications-bar/component.jsx]: There is no other calls to `NotificationsBar` except the one with `string` [here](https://github.com/OZhurbenko/bigbluebutton/blob/ab95c87f489982f9c9ce6827273a7ba39d20d78a/bigbluebutton-html5/imports/ui/components/notifications-bar/container.jsx#L56-L58)

**Comment by MaximKhlobystov** [comment]
Went through the initial PR's comments. Every single point seems to be fixed now. And the functionality works fine. Good job!

File: bigbluebutton-html5/imports/api/2.0/slides/server/modifiers/addSlide.js [hunk 2]
```diff
@@ -93,6 +93,14 @@ export default function addSlide(meetingId, presentationId, slide) {
...
     .then(({ width, height }) => {
+      // there is a rare case when for a very long not-active meeting
+      // the presentation files just disappear
+      // in that case just set the whole calculatedData to undefined
+      if (!width && !height) {
+        modifier.$set.calculatedData = undefined;
+        return Slides.upsert(selector, modifier, cb);
+      }
+
       // pre-calculating the width, height, and vieBox coordinates / dimensions
```
Comment (Klauswk) [inline bigbluebutton-html5/imports/api/2.0/slides/server/modifiers/addSlide.js]: I know this is not your mistake, but there is a typo here, "vieBox", change to viewbox.

File: bigbluebutton-html5/imports/api/2.0/slides/server/modifiers/resizeSlide.js [hunk 3]
```diff
@@ -27,19 +27,19 @@ export default function resizeSlide(meetingId, slide) {
   // fetching the current slide data
   // and pre-calculating the width, height, and vieBox coordinates / sizes
```
Comment (Klauswk) [inline bigbluebutton-html5/imports/api/2.0/slides/server/modifiers/resizeSlide.js]: Same typo here, "vieBox" to viewBox.

File: bigbluebutton-html5/imports/ui/components/polling/component.jsx [hunk 4]
```diff
@@ -1,7 +1,9 @@
-import React from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import Button from '/imports/ui/components/button/component';
-import styles from './styles.scss';
 import { defineMessages, injectIntl } from 'react-intl';
+import injectWbResizeEvent from '/imports/ui/components/presentation/resize-wrapper/component';
+import styles from './styles.scss';
```
Comment (Klauswk) [inline bigbluebutton-html5/imports/ui/components/polling/component.jsx]: Remove the .scss extension

File: bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/pencil-draw-listener/component.jsx [hunk 5]
```diff
@@ -2,7 +2,11 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import styles from '../styles.scss';
```
Comment (Klauswk) [inline bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/pencil-draw-listener/component.jsx]: remove the .scss extension

File: bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/shape-draw-listener/component.jsx [hunk 6]
```diff
@@ -2,12 +2,17 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import styles from '../styles.scss';
```
Comment (Klauswk) [inline bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/shape-draw-listener/component.jsx]: Remove the scss extesion

File: bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/text-draw-listener/component.jsx [hunk 7]
```diff
@@ -2,6 +2,11 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import styles from '../styles.scss';
```
Comment (Klauswk) [inline bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/text-draw-listener/component.jsx]: Remove the .scss extesion

**Pull request merged**